### PR TITLE
Feature/aero messages

### PR DESCRIPTION
**Pull request by 0xlsca**
### Major changes
1. Added AeroMessages
For the moment, the library is included via reference to `Lib/AeroMessages/AeroMessages.dll`. Adding it via git submodules didn't work for me, so the addition of the `.dll` had to suffice.
It was compiled from within VisualStudio with no additional settings set.
2. Changed `Character/BaseController` and the `Generic` controller to use AeroMessages
Out of the existing, only the `BaseController.MovementInput` and `Generic.ScheduleUpdateRequest` endpoints have been transitioned but others will follow soon.
Extensions for `GamePacket` and `IAero` have been added to support unpacking to- and serialization from `IAero` instances.
3. Added `MovementStateContainer` 
This class helps with handling the `MovementState` and see whether there are any unknown values being sent by the client.
Might also serve as a general template for handling enums but this only because I don't know of any other way to effectively handle received unknown flags.

### Minor Changes
1. Fixed the individual start `.bat` to use the `net6.0` folders
2. Moved Character-related controllers and entities each to a dedicated folder
Mentioning this here so the following sections make sense from an outsider's perspective
3. Changed `Channel` to support sending `IAero` instances
The class has also seen a refactoring of the send-logic to make it more `DRY`.
4. Added SilentCLD's `EntityGuid` implementation
See https://gist.github.com/SilentCLD/881839a9f45578f1618db012fc789a71

I tested the changes before submitting this PR so I assume them to be working

### CODE REVIEW

File: UdpHosts/GameServer/Packets/GSS/Character/BaseController/MovementInput.cs [hunk 1]
```diff
@@ -50,10 +50,10 @@ public class MovementInput
         public Vector AimDirection;
 
         [Field]
-        public ushort UnkUShort3;
+        public ushort UnkUShort3; // JetpackEnergy
 
         [Field]
-        public ushort UnkUShort4;
+        public ushort UnkUShort4; // GroundTimePositiveAirTimeNegative
```
Comment (eXpl0it3r) [inline UdpHosts/GameServer/Packets/GSS/Character/BaseController/MovementInput.cs:57]: I assume this and other already switched files are obsolete now and could be removed, correct?

File: UdpHosts/GameServer/Entities/Character/MovementStateContainer.cs [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameServer.Entities.Character
+{
+    /// <summary>
+    /// Allows eased access to the several flags
+    /// </summary>
+    internal class MovementStateContainer
```
Comment (eXpl0it3r) [inline UdpHosts/GameServer/Entities/Character/MovementStateContainer.cs:12]: Is the "Container" suffix coming from Firefall naming?  
To me it's a bit confusing, as it's not really a container in the sense of a collection, but maybe that's just my C++ bias 😄

File: UdpHosts/GameServer/Entities/Character/Character.cs [hunk 3]
```diff
@@ -38,9 +26,9 @@ public Character(IShard owner, ulong eid) : base(owner, eid)
         public Quaternion Rotation { get; set; }
         public Vector3 Velocity { get; set; }
         public Vector3 AimDirection { get; set; }
-        public CharMovement MovementState { get; set; }
+        internal MovementStateContainer MovementStateContainer { get; set; } = new();
```
Comment (eXpl0it3r) [inline UdpHosts/GameServer/Entities/Character/Character.cs:29]: Might want to move it down below the public fields, given that it's internal now

**Review by eXpl0it3r — APPROVED** [review]
We can either merge it and apply the small fixes in my PR or fix them, up to you, but looks good in general 🙂

File: UdpHosts/GameServer/Entities/Character/MovementStateContainer.cs (hunk 2 above)
Comment (0xlsca) [reply UdpHosts/GameServer/Entities/Character/MovementStateContainer.cs:12]: The `Container` suffix was chosen as a generic suffix without any implicit semantics in mind.
In essence, the class' purpose is to contain flags and accessors, so `Container` seemed like the correct choice. 
If there's a more fitting term available, we should definitely switch to it, but `Collection` is definitely out of question imo, haha 😁

File: UdpHosts/GameServer/Packets/GSS/Character/BaseController/MovementInput.cs (hunk 1 above)
Comment (0xlsca) [reply UdpHosts/GameServer/Packets/GSS/Character/BaseController/MovementInput.cs:57]: Yes, but I've noticed some differences between `AeroMessages` and PIN.
Some known fields in PIN are unknown in `AeroMessages`, I'll build a list of files to check afterwards.
They can be removed without further concerns.

Comment (0xlsca) [reply UdpHosts/GameServer/Packets/GSS/Character/BaseController/MovementInput.cs]: Resolved in #[7d5c351](https://github.com/themeldingwars/PIN/pull/16/commits/7d5c351f5703e74b65770a39714cf1fc0f81753e)

File: UdpHosts/GameServer/Entities/Character/Character.cs (hunk 3 above)
Comment (0xlsca) [reply UdpHosts/GameServer/Entities/Character/Character.cs]: Resolved in #[3b38dac](https://github.com/themeldingwars/PIN/pull/16/commits/3b38dac1e11b9ef47dcd056f782b1363945bd3c4)

**Comment by 0xlsca** [comment]
> We can either merge it and apply the small fixes in my PR or fix them, up to you, but looks good in general 🙂

Tbh, I'd go with merging this one first and then resolving the produced conflicts in #15 😁

File: UdpHosts/GameServer/GameServer.csproj [hunk 4]
```diff
@@ -35,6 +35,10 @@
         </None>
     </ItemGroup>
 
+    <ItemGroup>
+      <Folder Include="Packets\GSS\Generic\" />
+    </ItemGroup>
```
Comment (eXpl0it3r) [inline UdpHosts/GameServer/GameServer.csproj:40]: Is this needed?

Comment (0xlsca) [reply UdpHosts/GameServer/GameServer.csproj]: Good find! No, removed it in #[75ba2c0](https://github.com/themeldingwars/PIN/pull/16/commits/75ba2c02057ae791f6781ece2764b24bc11c6b4b)

**Review by eXpl0it3r — APPROVED** [review]

**Comment by eXpl0it3r** [comment]
Thanks for integrating Aero! 🥳

**Pull request merged**